test/parallel/test-tls-env-extra-ca.js

```diff
@@ -9,14 +9,15 @@ if (!common.hasCrypto) {
 }
 
 const assert = require('assert');
+const fs = require('fs');
 const tls = require('tls');
+
 const fork = require('child_process').fork;
```

Comment:
I put it separate from the rest because of the `.fork` appended to the end. I think our test writing guidelines might be ambiguous on this situation.  ¯\\_(ツ)_/¯